### DISCUSSION TITLE
How to change the ScoopDir/ScoopGlobalDir location without uninstallation?

### DISCUSSION THREAD

**Discussion by serifold** [ScoopInstaller/Scoop#5590] [Help and Questions] [answered]
My current ScoopDir is located at 'D:\Program Files\Scoop', and ScoopGlobalDir is located at 'D:\Program Files\Scoop\apps'. For some reason, I want to change the ScoopDir to 'D:\Scoop', and the ScoopGlobalDir to 'D:\Scoop\apps'. But I don't want to uninstall Scoop and all the apps then reinstall, because many in-app settings can't be easily restored by `Scoop import`.
What I'm thinking is could I run `irm get.scoop.sh -outfile 'install.ps1'` `.\install.ps1 -ScoopDir 'D:\Scoop' -ScoopGlobalDir 'D:\Scoop\apps'`, then copy the D:\Program Files\Scoop folder to D:\Scoop, finally change these environment variables D:\Program Files\Scoop\... to D:\Scoop\... Are these measures right? 
Or, according to the [doc](https://github.com/ScoopInstaller/Install#readme):
![image](https://github.com/ScoopInstaller/Scoop/assets/80401151/7400857d-f5e9-484c-9ced-3c2979bd1f02)
Is just changing SCOOP and SCOOP_GLOBAL enough? But I didn't find SCOOP or SCOOP_GLOBAL environment variables in my PC, just some apps installed by Scoop's environment variables, such as 'GIT_INSTALL_ROOT': 'D:\Program Files\Scoop\apps\git\current'...
I'm not sure, what should I do to change the location without modifying current installed apps' data?

**Comment by HUMORCE** [comment, marked as answer]
https://github.com/ScoopInstaller/Scoop/blob/f93028001fbe5c78cc41f59e3814d2ac8e595724/lib/core.ps1#L1258-L1262

Change their locations and move files, modify $env:PATH entry `D:\Program Files\Scoop\shims` to `<new_shims_dir_path>` then execute `scoop reset *`.

1. Environment variables: `$env:SCOOP` | `$env:SCOOP_GLOBAL`
2. Scoop configuration: `scoop config ROOT_PATH <path_to_scoop_root_dir>` | `scoop config GLOBAL_PATH <path_to_global_apps_dir>` (Current situation of your scoop installation)
3. Default Location: `%USERPROFILE%\scoop` | `%PROGRAMDATA%\scoop`

**Reply by serifold** [reply]
❤Thank you very much, it works, and I learned more about Scoop configuration at the same time.
